packages/plugin-twitter/src/actions/post.ts

```diff
@@ -6,115 +6,67 @@ import {
     composeContext,
     elizaLogger,
     ModelClass,
-    formatMessages,
-    generateObject,
+    generateText,
+    stringToUuid,
 } from "@elizaos/core";
-import { Scraper } from "agent-twitter-client";
 import { tweetTemplate } from "../templates";
-import { isTweetContent, TweetSchema } from "../types";
 
-async function composeTweet(
-    runtime: IAgentRuntime,
-    _message: Memory,
-    state?: State
-): Promise<string> {
-    try {
-        const context = composeContext({
-            state,
-            template: tweetTemplate,
-        });
-
-        const tweetContentObject = await generateObject({
-            runtime,
-            context,
-            modelClass: ModelClass.SMALL,
-            schema: TweetSchema,
-            stop: ["\n"],
-        });
+function truncateToCompleteSentence(text: string, maxLength: number): string {
```

Comment:
This code is copied from client-twitter, right? Maybe we could move them to a utils file for better reusability. :)

Reply:
thats sounds cool 🫡